app/javascript/test/cell.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Cell from '../components/Cell';
+
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import './setupTests';
+
+
+describe('Test Cell component', () => {
+    it('Test click event', () => {
```

Comment:
Can you come up with more descriptive test names? For example, `it('plays a game move')`.